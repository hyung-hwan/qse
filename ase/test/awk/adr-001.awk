#!/bin/awk

BEGIN { 
        RS = "\n\n"; 
        FS = "\n"; 
}
{ 
	print $1, $NF;         
}
