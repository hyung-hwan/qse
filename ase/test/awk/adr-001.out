BEGIN {
	RS = "\n\n";
	FS = "\n";
}

{
	print $1,$NF;
}

James Brown 012-345-678
Richie Ren 02-3473-9192
Toh WeeKung 9102-1203
